README.rst

```diff
@@ -13,12 +13,15 @@ Tribler
 
 *Towards making Bittorrent anonymous and impossible to shut down.*
 
-We use our own dedicated Tor-like network for anonymous torrent downloading. We implemented and enhanced the *Tor protocol specifications* plus merged them with Bittorrent streaming. More info: https://github.com/Tribler/tribler/wiki
-Tribler includes our own Tor-like onion routing network with hidden services based seeding and end-to-end encryption, detailed specs: https://github.com/Tribler/tribler/wiki/Anonymous-Downloading-and-Streaming-specifications
+We use our own dedicated Tor-like network for anonymous torrent downloading.
+We implemented and enhanced the `Tor protocol specifications <https://github.com/Tribler/tribler/wiki>`_.
```

Comment:
Maybe we can link to [this page](https://github.com/Tribler/tribler/wiki/Anonymous-Downloading-and-Streaming-specifications)?

Reply:
Changed.